std/testing/bench_example.ts

```diff
@@ -26,4 +26,4 @@ bench(function throwing(b): void {
 });
 
 // Bench control
-runIfMain(import.meta, { skip: /throw/ });
+runBenchmarks({ skip: /throw/ });
```

Comment:
```suggestion
if (import.meta.main) {
  runBenchmarks({ skip: /throw/ });
}
```
Also, the `runIfMain()` section in `std/testing/README.md` needs to be removed.

Reply:
Oops, sorry, missed that.